package.json

```diff
@@ -34,8 +34,8 @@
     "node": ">=0.10.x"
   },
   "main": "component/index.js",
-  "dependencies": {
-    "react": "^0.13.1"
+  "peerDependencies": {
+    "react": ">= 0.13.1, < 0.15.0"
```

Comment:
`"react": ">= 0.13.1, < 0.15.0"` does NOT match with `0.14.0-rc1`.

It should be something similar to `"react": ">=0.13.1 || ^0.14.0-rc1"`